autogpt_platform/db/docker/docker-compose.yml

```diff
@@ -7,10 +7,15 @@
 
 name: supabase
 
+x-envfile: &env_file
+  env_file:
+    - .env
+
 services:
 
   studio:
     container_name: supabase-studio
+    <<: *env_file
```

Comment:
Does this make the entries below like `POSTGRES_PASSWORD: ${POSTGRESS_PASSWORD}` redundant?